src/features/Dashboard/components/MembershipPanel.tsx

```diff
@@ -16,6 +16,7 @@ import { flatten, uniq } from '@vendor';
 
 const SDashboardPanel = styled(DashboardPanel)<{ isMemberOrExpired: boolean }>`
   display: flex;
+  height: 171px;
```

Comment:
Fixed height creates vertical overflow in mobile view

<img width="380" alt="Screenshot 2021-01-25 at 13 43 37" src="https://user-images.githubusercontent.com/2429708/105707505-5630fb00-5f13-11eb-9754-d8cc84b18319.png">